bin/ghe-backup

```diff
@@ -200,10 +200,15 @@ is_binary_backup(){
 
 echo "Backing up MySQL database ..."
 bm_start "ghe-export-mysql"
-echo 'set -o pipefail; ghe-export-mysql | gzip' |
+export_command="ghe-export-mysql"
+if ! is_binary_backup; then
+  # binary backup is already compressed
+  export_command+=" | pigz"
+fi
+echo "set -o pipefail; ${export_command}" |
```

Comment:
I don't think we need `${var}` interpolation here.  https://github.com/github/backup-utils/blob/master/STYLEGUIDE.md#use-var-for-interpolation-only-when-required has more info on the consistency we've tried to keep.

Reply:
fixed, thanks!